picard/script.py

```diff
@@ -307,6 +308,22 @@ def register_script_function(function, name=None, eval_args=True,
     )
 
 
+def _compute(operation, operands):
+
+    operand_list = [operands.get('x'), operands.get('y')] + list(operands.get('args'))
+    int_operators = {'add': operator.add,
+                     'sub': operator.sub,
+                     'mul': operator.mul,
+                     'div': operator.div,
+                     'mod': operator.mod}
+    logic_operators = {'and': (lambda x, y: x and y),
+                       'or': (lambda x, y: x or y)}
```

Comment:
`int_operators` and `logic_operators` are constant dicts, right? I would move them outside the function, no need to re-evaluate them on each run.